test/e2e/wdio.bs.conf.js

```diff
@@ -1,7 +1,12 @@
+const execSync = require('child_process').execSync
+const path = require('path')
+
 const baseConf = require('./wdio.base.conf')
 const browsers = require('./browsers.conf')
 const getBuildInfos = require('../getBuildInfos')
 
+const ipAddress = execSync(path.join(__dirname, '../..', 'scripts/get-ip.sh'))
```

Comment:
What about:
```js
const ipAddress = Array.from(
  execSync("ifconfig", { encoding: "utf-8" })
    .matchAll(/inet (?:addr:)?((?:[0-9]*\.){3}[0-9]*)/g),
  match => match[1]
).find(ip => ip !== '127.0.0.1')
```

Reply:
any advantage to use node instead of bash?

Reply:
Readability and portability. BTW, this may be even simpler:
```js
const ipAddress = Object.values(os.networkInterfaces())
  .flat()
  .find(({ family, internal }) => family === 'IPv4' && !internal)
  .address
```